src/System.Private.CoreLib/src/System/Enum.CoreCLR.cs

```diff
@@ -146,36 +146,5 @@ private static RuntimeType ValidateRuntimeType(Type enumType)
                 throw new ArgumentException(SR.Arg_MustBeType, nameof(enumType));
             return rtType;
         }
-
-        public int CompareTo(object? target)
-        {
-            const int retIncompatibleMethodTables = 2;  // indicates that the method tables did not match
-            const int retInvalidEnumType = 3; // indicates that the enum was of an unknown/unsupported underlying type
-
-            if (this == null)
-                throw new NullReferenceException();
-
-            int ret = InternalCompareTo(this, target);
```

Comment:
Could delete the `InternalCompareTo` method declaration from this file also.

Reply:
https://github.com/dotnet/coreclr/pull/27837 Thanks!